src/utils/css/matrix.js

```
@@ -0,0 +1,17 @@
+/**
+ * Format a CSS transform matrix with the given values.
+ *
+ * @param  {Number} options.scaleX     The scale on the x axis.
+ * @param  {Number} options.scaleY     The scale on the y axis.
+ * @param  {Number} options.skewX      The skew on the x axis.
+ * @param  {Number} options.skewY      The skew on the y axis.
+ * @param  {Number} options.translateX The translate on the x axis.
+ * @param  {Number} options.translateY The translate on the y axis.
+ * @return {String}                    A formatted CSS matrix transform.
+ */
+export default function matrix(t) {
+  // eslint-disable-next-line no-param-reassign
+  t = t || {};
+  return `matrix(${t.scaleX || 1}, ${t.skewX || 0}, ${t.skewY || 0}, ${t.scaleY ||
```

Should we test that `t` is an object before returning ? 

And throw an error message that says something like : 
`matrix(param) must an object formated like so : { scaleX: value, skewX: value, skewY: value, scaleY: value, translateX.value, translateY: value }`

That might also allow to incorporate a more robust jest test ?

---

This method might seem a little "dirty" without tests and errors, but it is so by design to keep it performant as it might be used in `requestAnimationFrame` loops.

See [https://jsben.ch/fRslP](https://jsben.ch/fRslP) for the large difference between the one with test and the one without.

---

Passing a value that is not an object will fail silently and return the default values, I will update the Jest test with this case.

---

The perf difference is indeed huge ! 
I guess a good documentation on how to use the [`matrix()`](http://gph.is/2lEdjWA), will prevent most errors